From 1bc62850c525eed947431fff0470232d772884de Mon Sep 17 00:00:00 2001
From: Keerthana K <keerthanak@vmware.com>
Date: Fri, 18 Nov 2022 07:13:06 +0000
Subject: [PATCH 2/2] stalld: fix /bin/bash

bash rpm provides only /bin/bash and /bin is a symbolic
link to /usr/bin

Signed-off-by: Keerthana K <keerthanak@vmware.com>
---
 scripts/throttlectl.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/throttlectl.sh b/scripts/throttlectl.sh
index a01ba14..4737bee 100755
--- a/scripts/throttlectl.sh
+++ b/scripts/throttlectl.sh
@@ -1,4 +1,4 @@
-#!/usr/bin/bash
+#!/bin/bash
 
 # This script is called to either turn off or turn on RT throttling
 # The 'off' argument causes the current values of the throttling
-- 
2.28.0

